int64 robort_id
---
int64 remaining_bullet
